server/models/user-model.ts

```diff
@@ -39,4 +40,8 @@ export default class User {
   get passwordHash(): string {
     return this.#passwordHash;
   }
+
+  toToken(): UserToken {
```

Comment:
I'm curious about the name of this method. I feel like I would expect this to return a token similar to the format we're used to seeing with JWT tokens, but the return statement is clearly something different. Is this just something you've designated as a token within the scope of your code?